app/views/comments/_comments.html.slim

```diff
@@ -1,6 +1,6 @@
 -if user_signed_in?
-  = render 'comments/form', commentable: commentable \
-  unless current_user.banned?
+  - if active?
```

Comment:
@skcc321, yes, he can. Inactive user means banned user. This line just hides comment fields from banned users.